### PR TITLE
gauge-settings

### PR DESCRIPTION
**Pull request by kghamilton89**
Fairly straightforward refactoring to bring Gauge Settings under the jurisdiction of our new 5695 Style Rules.
Rules are all specific to Gauge Widget. The settings in question are shown below:
* `format-tip`
* `format-counter`
Each of the example configurations for these features the following style of syntax: `format-tip = d3.format("0.1f")` which Sasha described as sort of a hack. How should we approach documenting such syntax or should we exclude it from the docs entirely as we can achieve the same effects using our Label Formatting syntax?

### CODE REVIEW

File: widgets/gauge-chart/README.md [hunk 1]
```diff
@@ -1,64 +1,81 @@
...
+[![](../../images/button.png)](https://apps.axibase.com/chartlab/d84140e2)
+
+## Widget Settings
+
+* The settings apply to the `[widget]` section
+* [Shared](../shared/README.md) `[widget]` settings are inherited
+
+Name | Example | Description | &nbsp;
+:--|:--|:--|:--
+|[`mode`](#mode)|`mode = sleek`|Widget mode.<br>Possible values: `auto`, `sleek`, `half`.<br>Default value: `auto`|[↗](https://apps.axibase.com/chartlab/d7016bd7)|
```
[the remark this answers is not in the document]
Comment (kghamilton89) [reply widgets/gauge-chart/README.md]: I was just trying to maintain consistency with other settings which feature `auto`.

File: widgets/gauge-chart/README.md [hunk 2]
```diff
@@ -1,64 +1,81 @@
...
+|[`format-axis`](#format-axis)|`format-axis = Math.round(value * 10)/10 + "%"`|Value axis format.<br>Refer to [Label Formatting](../../syntax/label-formatting.md) for possible values.|[↗](https://apps.axibase.com/chartlab/01288425/29/)|
+|[`format-counter`](#format-counter)|`format-counter = d3.format("0.3f")`|Counter format.<br>Refer to [Label Formatting](../../syntax/label-formatting.md) for possible values.|[↗](https://apps.axibase.com/chartlab/01288425/29/)|
+|[`ring-width`](#ring-width)|`ring-width = 0.2`|Width of the gauge ring area.<br>Percent of total possible width.<br>Possible values:<br>**Inclusively** between `0` and `1`.|[↗](https://apps.axibase.com/chartlab/3134e9d6)|
+|[`border-width`](#border-width)|`border-width = 0.3`|Offset between gauge ring and container.<br>Percent of total possible radius.<br>Possible values:<br>**Inclusively** between `0` and `1`.|[↗](https://apps.axibase.com/chartlab/3134e9d6/2)|
+|[`pin-radius`](#pin-radius)|`pin-radius = 0.2`|Radius of pin base.<br>Percent of total possible radius.<br>Possible values:<br>**Inclusively** between `0` and `1`.|[↗](https://apps.axibase.com/chartlab/ea2d99cf)|
```
[the remark this answers is not in the document]
Comment (kghamilton89) [reply widgets/gauge-chart/README.md]: changed

**Comment by kghamilton89** [comment]
Requested changes implemented.

**Comment by kghamilton89** [comment]
Document completion is waiting for an answer on whether or not we should include `format-tip = d3.format("0.1f")` type syntax.

**Comment by rodionos** [comment]
Please review all PRs and remove Boolean expression if the setting supports fixed boolean values, sich as 'display-tip'.

**Comment by rodionos** [comment]
If the setting support only true or false (and not a boolean expression), lets document it as:

```
Possible values: `false`, `true`.
```

**Comment by kghamilton89** [comment]
* Refactoring of content.
* Removal of "Boolean expression" terminology.
* Addition of "Possible values: `false`, `true`."
* Refactoring of example configurations to isolate target settings.

**Comment by kghamilton89** [comment]
Build is failing due to real links being read as dead (relative links). I mentioned it to AR in another PR.

**Comment by kghamilton89** [comment]
This PR is a complete refactor (not just format, but content as well).

**Comment by kghamilton89** [comment]
All charts examples have been passed through the Charts plugin as well and verified for accuracy.

**Comment by rodionos** [comment]
Update the chart plugin. The plugin now raises alert if the number of colors is not the number of thresholds minus 1. Re-instate original descriptions.

As a result colors and threshold descriptions need to be revised.

**Comment by kghamilton89** [comment]
* Table updated to meet new standards.
* `thresholds` setting updated to describe new behavior.

**Pull request merged**